docker/images/gcs-emulator/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM fsouza/fake-gcs-server:latest
```

Comment:
This is tricky. I have definitely run into cases where someone builds things after I do and we end up with different versions of things and then it takes a while to figure out that whatever problem we're seeing is due to us having two different versions of things.

Can we switch this to 1.47.8?

https://hub.docker.com/layers/fsouza/fake-gcs-server/1.47.8/images/sha256-e1a52a152250dc424bc328f01f859aeb4b3b04f9e733900f2f7fcf7238972d13?context=explore